text/0517-io-os-reform.md

```diff
@@ -1514,7 +1599,7 @@ For the current `addrinfo` module:
 
 For the current `ip` module:
 
-* The `ToSocketAddr` trait should become `ToSocketAddrs`
+* The `ToInetAddr` trait should become `ToInetAddrs`
```

Comment:
There doesn't seem to be an existing `ToInetAddr` in rust at the moment, perhaps find-and-replace went a bit too far here?